targets/frontend/src/components/editor/CodeEditor.js

```diff
@@ -1,8 +1,9 @@
-import "ace-builds/src-noconflict/mode-json";
-import "ace-builds/src-noconflict/theme-github";
-
+/* eslint-disable simple-import-sort/imports */
```

Comment:
It could be great to explain why the rule is disabled here. Perhaps adding a comment:

`// Imports order are important and can't be in alphabetical order.`

To avoid to remove this line mistakenly.